development-docs/isExternalLibraryImport.md

```diff
@@ -0,0 +1,63 @@
+## How we override isExternalLibraryImport
```

Comment:
This file is notes to myself and any other ts-node developers.  Given the audience, it's not as polishes as the rest of ts-node, and I can remove it if needed.